src/main/java/com/faforever/client/fx/FxmlLoaderImpl.java

```diff
@@ -25,12 +27,17 @@
   UiService uiService;
   @Inject
   ApplicationContext applicationContext;
+  @Inject
+  I18n i18n;
```

Comment:
like this:
`  @Inject
  public FxmlLoaderImpl(I18n i18n)
  {
    this.i18n=i18n;
  }`